### ISSUE TITLE
hello

### ISSUE THREAD

**Issue by depcco** [divelab/GOOD#31]
I read your paper and found Table 1. Now I have some questions. I noticed that the concept shift has a higher value than the covariate shift, but in my opinion, the concept shift is a harder task. Could you explain why?

![image](https://github.com/divelab/GOOD/assets/169051868/06af6dbf-f4a7-4a57-8485-0bc9f3796972)

**Comment by depcco** [comment]
Is there a difference in difficulty between covariate shift and concept shift within the same dataset? I hope to get your explanation.

**Comment by CM-BF** [comment]
Hi @depcco,

It is true that concept shift is harder than covariate shift from some perspectives, but the performances are not comparable on difference shifts. The performance drop indicates the sensitivity of the model given a specific shift, but does not reflect the difficulty of the shifts. For example, models cannot perform well under covariate shift, but it does not mean that covariate shift is hard to solve vice versus.

Best,
Shurui

**Comment by depcco** [comment]
Thank you very much, I have gained a lot of valuable knowledge from our discussions. I hope you can help me with another question regarding experimental data.I noticed that after the final training is completed, it outputs a "Loading best Out-of-Domain Checkpoint number" and a "Loading best in-Domain Checkpoint number".  I would like to know which output results the data IDs IDID and OODOOD in the experimental data table correspond to. Also, i read you another paper-[LECI](https://github.com/divelab/LECI)，are the differences between different exp_round solely due to hyperparameters?

<img width="362" alt="1717081085151" src="https://github.com/divelab/GOOD/assets/169051868/e70e2940-5fc2-467a-8001-6705a78a40b5">

**Comment by CM-BF** [comment]
> I would like to know which output results the data IDs IDID and OODOOD in the experimental data table correspond to

IDID uses the In-Domain checkpoint. OODOOD uses the Out-of-Domain checkpoint.

> are the differences between different exp_round solely due to hyperparameters?

The difference mainly attributes to the change of random seeds.

**Comment by depcco** [comment]
@CM-BF 
Thank you for your response. However, when running LECI, I noticed that the data varies significantly between different exp_rounds. For example, in the fourth round, the OODOOD value is 41.47, whereas in the ninth round, the OODOOD value is 72.33. I would like to know if this is a normal situation. Did you take the average of the data over ten rounds or select the best round for the experimental data in your article? The data I reproduced comes from GOOD motif，size。Looking forward to your reply.    
![image](https://github.com/divelab/GOOD/assets/169051868/9a64618b-c043-4017-b13b-20bb57fba5be)
<img width="519" alt="1717124401399" src="https://github.com/divelab/GOOD/assets/169051868/939e726e-e1d6-4d8d-9ce1-e4a8a0a540f6">
